.python-version

```diff
@@ -0,0 +1 @@
+3.7.6
```

Comment:
is this a standard? I love this

Reply:
pyenv! It's like tfenv only even cooler.